source/install/cloud-changelog.md

```diff
@@ -4,12 +4,41 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-09-15](#release-2022-09-15)
 - [Release 2022-09-08](#release-2022-09-08)
 - [Release 2022-09-01](#release-2022-09-01)
 - [Release 2022-08-25](#release-2022-08-25)
 - [Release 2022-08-18](#release-2022-08-18)
 - [Release 2022-08-10](#release-2022-08-10)
-- [Release 2022-08-03](#release-2022-08-03)
+
+## Release 2022-09-15
+
+### Highlights
+
+#### Notify Admin v2
+ - Added more context to the “Notify admin” feature to help Admins, such as who asked to upgrade, why they requested the upgrade, and how many people requested it.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a new Top Playbooks Insights widget.
```

Comment:
I believe there are a total of 4 Insights widgets being introduced in v7.3:  CC @laneycs 
- Top Playbooks
- Most active DMs (My insights only)
- Least active channels
- Newest team members

Reply:
Top Playbooks is releasing in this cloud release because the pre-packaged playbooks version that supports it wasn't merged until this week. The other Insights widgets already shipped in previous cloud releases.